Common/File/FileUtil.cpp

```diff
@@ -563,6 +650,15 @@ bool Move(const Path &srcFilename, const Path &destFilename) {
 // Returns the size of file (64bit)
 // TODO: Add a way to return an error.
 uint64_t GetFileSize(const Path &filename) {
+	if (Android_IsContentUri(filename.ToString())) {
```

Comment:
Should we be checking the type here?

-[Unknown]

Reply:
Rewrote it.